.gitlab-ci.yml

```diff
@@ -501,15 +533,13 @@ build/amd/openmpi/clang/rocm502/release/shared:
   extends:
     - .build_and_test_template
     - .default_variables
-    - .full_test_condition
-    - .use_gko-rocm502-openmpi-gnu11-llvm11-multi-gpu
+    - .quick_test_condition
+    - .use_gko-rocm502-openmpi-gnu11-llvm11
   variables:
     C_COMPILER: "clang"
     CXX_COMPILER: "clang++"
     BUILD_OMP: "ON"
     BUILD_HIP: "ON"
-    BUILD_MPI: "ON"
```

Comment:
mistake?

Reply:
No, that was a job on the amdci machine, where we do not want MPI jobs anymore. I have updated the job name accordingly now.